conf/st2.conf.sample

```diff
@@ -100,6 +100,16 @@ logging = conf/logging.exporter.conf
 # Directory to dump data to.
 dump_dir = /opt/stackstorm/exports/
 
+[garbagecollector]
```

Comment:
It would be nice if we could control starting this process based on whether a config is enabled. Are we going to offload to this init systems? By virtue of whether service is enabled or disabled, this will run/not run?